packages/gestalt/src/Card.flowtest.js

```diff
@@ -4,5 +4,5 @@ import Card from './Card.js';
 
 const Valid = <Card />;
 
-// $FlowExpectedError[prop-missing] Cannot create `Card` element because property `nonexisting` is missing in  props.
+// $FlowExpectedError[prop-missing]
```

Comment:
Just wondering why remove the additional info from the comments here?

Reply:
@jennyscript will be too cumbersome to maintain - I looked at flow-typed where they also moved away from adding these extra comments.